evidence.go

```diff
@@ -46,6 +46,16 @@ func (e *Evidence) GetInstanceID() *[]byte {
 	return &instID
 }
 
+// GetImplementationID returns the ImplementationID from PSA token
+// or a nil pointer if no suitable ImplementationID could be located.
```

Comment:
```suggestion
// GetImplementationID returns the ImplementationID claim from the
// PSA token or a nil pointer if no suitable ImplementationID could
// be located.
```

also, small question: is it possible that the ImplID is not set at this point?